### ISSUE TITLE
Filters and sorting blink once/twice when opening a page with it.

### ISSUE THREAD

**Issue by greenthus** [vmware-clarity/ng-clarity#127]
![image](https://user-images.githubusercontent.com/104067056/170044753-18aeef82-132d-4c74-ba53-0bde07de82fa.png)

or some other solution that shows... notice you have a sorting/filtering ON.
![image](https://user-images.githubusercontent.com/104067056/170045066-ef08edbb-5e3c-4a95-9dd4-397259374153.png)

other solution idea 2:
https://dribbble.com/shots/3723001-29-Focus

**Comment by Jinnie** [comment]
Any solution in this area should be synced with the accessibility team, to make sure that is usable by users with disabilities, using screenreader, magnification, etc.

**Comment by greenthus** [comment]
yeah that makes sense Jinnie.

**Comment by greenthus** [comment]
can it be made that elements have their unique class and or a ID automatically unique attached?
![image](https://user-images.githubusercontent.com/104067056/171859100-efc39227-c43d-4e07-9a24-47d6b645ed3f.png)

also for the (sorting/filtering) icons would be nice.
